### PR TITLE
Budget App: Feature

### PR DESCRIPTION
**Pull request by HtetWaiYan7191**
### Setting up

- Set up a new rails app with postgres development
- Install linter and rubocop
- Set up model and associations
- Set up routes 
- Use devise for authentication purposes 

### Pages and functionality
-  Add splash page for login and sign up 
- Add login page
-  Add sign up page
- Implement categories page and its CRUD operations
- Implement transaction page and it's CRUD operations
- Implement required integration tests and model.

### Presentation
- Add live demo link and video walkthrough live through 

Thanks  for reviewing my code 😁 Have a great day ⭐

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +56,21 @@
...
+- [Live Demo Link](https://budget-app-ugv5.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Video Walkthrough -->
+
+## 🎬  VIdeo Walkthrough <a name="live-demo"></a>
+
+- [Video Demo Link](https://www.loom.com/share/fbe444803b064d86bb195fb783c96295?sid=80d8f963-e415-4115-a1b9-b458c9e735ad)
+
```
Comment (uwadonat) [inline README.md]: - [x] You did an amazing presentation. As requested in the project requirements, kindly, let's add in you video a highlight about a piece of the code that you find interesting or are proud of making. Pass to your code editor and show.

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @HtetWaiYan7191 

You did an amazing job.
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Your commit messages are very descriptive ✔️
- Detailed readme file ✔️
- Well organized and clean project ✔️
- Correct workflow was used ✔️

almost

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (HtetWaiYan7191) [reply README.md]: Thanks sir . I will did that 😁

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @HtetWaiYan7191, 




68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Requested change well implemented🔥 
✅ Professional README file 💯 
✅ Good commit messages 👍 


## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**